h2o-py/h2o/model/metrics_base.py

```diff
@@ -119,7 +119,7 @@ def show(self):
             print("AUC: " + str(self.auc()))
             print("pr_auc: " + str(self.pr_auc()))
             print("Gini: " + str(self.gini()))
-            self.confusion_matrix().show()
+            # self.confusion_matrix().show() // TODO: confusion matrix not yet converted, commented out temporarily
```

Comment:
careful here :)

Reply:
Will be removed, as this is no longer needed. The solution we have is better.